model-optimizer/extensions/ops/scatternd.py

```diff
@@ -57,14 +57,16 @@ def infer(node: Node):
         # 1. ranks of both input and indices must be at least 1
         assert len(input_shape) >= 1 and len(indices_shape) >= 1, \
             'The node "{}" input and indices ranks must be at least 1'.format(node_name)
-           
+
         # 2. the last dimension of indices shape must be at most a rank of input
         assert indices_shape[-1] <= len(input_shape), \
             'The last dimension of indices shape must be at most a rank of input for the node "{}"'.format(node_name)
 
         # 3. updates is a tensor of shape indices_shape[:-1] + input_shape[indices_shape[-1]:]
+        # if expected updates shape is scalar, updates can be tensor of shape [1]
```

Comment:
updates can be tensor with the single element (for example, of shape [1], [[1]], etc.)